qlib/utils/__init__.py

```diff
@@ -789,7 +792,9 @@ def lazy_sort_index(df: pd.DataFrame, axis=0) -> pd.DataFrame:
     """
     idx = df.index if axis == 0 else df.columns
     # NOTE: MultiIndex.is_lexsorted() is a deprecated method in Pandas 1.3.0 and is suggested to be replaced by MultiIndex.is_monotonic_increasing (see discussion here: https://github.com/pandas-dev/pandas/issues/32259). However, in case older versions of Pandas is implemented, MultiIndex.is_lexsorted() is necessary to prevent certain fatal errors.
-    if idx.is_monotonic_increasing and not (isinstance(idx, pd.MultiIndex) and not idx.is_lexsorted()):
+    if idx.is_monotonic_increasing and not (
```

Comment:
Hi, 
Do you think this will make the logic easier to understand?
```python
if not idx.is_monotonic_increasing or \   # This case is for both new and old version
        not is_deprecated_lexsorted_pandas and isinstance(idx, pd.MultiIndex) and not idx.is_lexsorted(): # this case is for the old version
    return df.sort_index(axis=axis)
else:
    return df
```

Reply:
LGTM. The black will change the code to below.
   ```
 if (
        not idx.is_monotonic_increasing
        or not is_deprecated_lexsorted_pandas
        and isinstance(idx, pd.MultiIndex)
        and not idx.is_lexsorted()
    ):  # this case is for the old version
        return df.sort_index(axis=axis)
    else:
        return df

```